File: src/classes/Endereco.java
package classes;

public class Endereco {
	private int id;
	private int numero;
	private String rua;
	private String bairro;
	private String cidade;
	private String estado;
	private Parceiro conta;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	
	public int getNumero() {
		return numero;
	}
	public void setNumero(int numero) {
		this.numero = numero;
	}
	
	public String getRua() {
		return rua;
	}
	public void setRua(String rua) {
		this.rua = rua;
	}
	
	public String getBairro() {
		return bairro;
	}
	public void setBairro(String bairro) {
		this.bairro = bairro;
	}
	
	public String getCidade() {
		return cidade;
	}
	public void setCidade(String cidade) {
		this.cidade = cidade;
	}
	
	public String getEstado() {
		return estado;
	}
	public void setEstado(String estado) {
		this.estado = estado;
	}
	
	public Parceiro getConta() {
		return conta;
	}
	public void setConta(Parceiro conta) {
		this.conta = conta;
	}

}

File: API C#/ListaPublica/ListaPublica/Classes/Endereco.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ListaPublica.Classes
{
    public class Endereco
    {
        public int id { get; set; }
        public int numero { get; set; }
        public string rua { get; set; }
        public string bairro { get; set; }
        public string cidade { get; set; }
        public string estado { get; set; }
        public string nome { get; set; }
        //public Parceiro parceiro { get; set; }
    }
}
